overview/views.py

```diff
@@ -166,6 +166,51 @@ def get_context_data(self, *args, **kwargs):
         return context
 
 
+# a specific "spreadsheet-like" view of candidates' biking support
+class CandidateBikingList(ListView):
+    model = Candidate
+    template_name = 'overview/candidates_biking.html'
+
+    def get_context_data(self, *args, **kwargs):
+        context = super(CandidateBikingList, self).get_context_data(*args, **kwargs)
+
+        candidates = Candidate.objects.exclude(hide=True).exclude(is_running=False).order_by("fullname").prefetch_related("endorsements")
```

Comment:
```suggestion
        candidates = Candidate.objects.exclude(hide=True).exclude(is_running=False).order_by("fullname")
```

In endorsed_by_group, you have a .filter and .exists but prefetch related only works with .all so that method would be written as
```
for org in self.endorements.all():
    if org.name == org_name:
        return True
return False
```